hardware/raspberrypi/bootmodes/net.md

```diff
@@ -1,5 +1,7 @@
 # Network booting
 
+**Network booting is available on the Raspberry Pi 3B, 3B+, 3A+ and Raspberry Pi 2B v1.2 models only.**
```

Comment:
The 3A+ doesn't have a built-in ethernet  adaptor... :wink:

Reply:
Sorry - I'm getting confused between device boot and host boot I think.

Reply:
Copy and paste error, combined with the fact I failed to remember the fact you've just pointed out. Fixed.